vnext/Microsoft.ReactNative/IReactDispatcher.h

```diff
@@ -42,8 +42,8 @@ struct ReactDispatcher : implements<ReactDispatcher, IReactDispatcher, Mso::Reac
   static IReactPropertyName JSDispatcherIdleWaitStartingEventName() noexcept;
   static IReactPropertyName JSDispatcherIdleWaitCompletedEventName() noexcept;
 
-  void Post(Mso::DispatchTask &&task) const noexcept override;
-  void InvokeElsePost(Mso::DispatchTask &&task) const noexcept override;
+  void Post(Mso::DispatchTask &&task) const noexcept;
```

Comment:
Is this supposed to override (implement) IDispatchQueue2::Post() and IDispatchQueue2::InvokeElsePost()? If so, the 'override' keyword appears correct. What is clang complaining about?

Reply:
I think `InvokeElsePost()` was removed by mistake. clang claims that there is no method to override.

Reply:
```
stderr: In file included from react-native-windows\vnext\Microsoft.ReactNative\XamlTypeInfo.g.cpp:16:
react-native-windows\vnext\Microsoft.ReactNative/IReactDispatcher.h:45:54: error: only virtual member functions can be marked 'override'
  void Post(Mso::DispatchTask &&task) const noexcept override;
                                                     ^~~~~~~~
react-native-windows\vnext\Microsoft.ReactNative/IReactDispatcher.h:46:64: error: only virtual member functions can be marked 'override'
  void InvokeElsePost(Mso::DispatchTask &&task) const noexcept override;
                                                               ^~~~~~~~
```

Clang does not think these are virtual methods.

Reply:
I don't see any warnings or errors when compiling with MSVC without the override either.

Reply:
Yes I think the override isn't needed.  winrt::implements ensures the interface is correct without the override.